doc/meson.build

```diff
@@ -19,7 +19,10 @@ gen_man_pages = []
 
 syms = result.stdout().strip().split('\n')
 foreach sym : syms
-  gen_man_pages += ['@0@.3'.format(sym)]
+  # for consistency with autotools build and because we don't pass -v to c2man
+  if sym != 'fribidi_unicode_version' and sym != 'fribidi_version_info'
```

Comment:
The autotools build just skips the last 2 lines, you could do the same here for consistency.

Reply:
One *could* implement that as well, but ... do we really have to copy every atrocity in the autotools build just for internal consistency? This is much clearer and less confusing IMHO :)